motd - updates...

IcDasNMs opened the pull request:
changes until 2026-01-04...
umk3 [mgoumkarg]
sfa3, snes_dbzbuto2, snes_dbzbuto2ta, xmcota, msh, xmvsfu, wh1 [lcfreezer]
flycast_cvs2, ssf2xjr1, sfa2u, sfa3 [kaisark]
snes_yuyufin, flycast_ngbc, samsh5sp`*` [dojoesports]
lastbld2 [hambaaga]
kchampvs [nihyru]
jojobanr1`*` [blacksavageman]
sfiii3nr1 [famefgc, tremolo3]
sfa3 [kyokuji]
vsavj [h0llowsid]
flycast_gram2000 [crowspaceboy]

`*`link removal

IcDasNMs committed:
motd updates until 2025-11-08

umk3 [mgoumkarg]
sfa3 [lcfreezer, kaisark]
snes_yuyufin [dojoesports]
flycast_ngbc [dojoesports]
flycast_cvs2 [kaisark]
ssf2xjr1 [kaisark]
sfa2u [kaisark]

IcDasNMs committed:
changes until 2025-11-11

snes_dbzbuto2 [lcfreezer]
snes_dbzbuto2ta [lcfreezer]

IcDasNMs committed:
changes until 2025-11-15

xmcota [lcfreezer]

IcDasNMs committed:
changes until 2025-12-06

msh [lcfreezer]

nihyru commented:
{
  "gameid": "kchampvs",
  "motd": "Discord: https://discord.gg/CN9NVGGgJz"
}

IcDasNMs committed:
changes until 2026-01-04

samsh5sp [dojoesports] - link removal
lastbld2 [hambaaga]
xmvsf/xmvsfu/xmvsfjr3/wh1 [lcfreezer]
kchampvs [nihyru]

IcDasNMs committed:
Update motd.json

flycast_vtennis, flycast_vtennis2 [mgoumkarg]

IcDasNMs committed:
changes until 2025-01-25

samsho2 [tommytamsworth]

IcDasNMs committed:
changes until 2026-02-01

mshvsf [lcfreezer]

IcDasNMs committed:
changes until 2026-04-01

jojobanr1: removed 3 links [blacksavageman]
sfiii3nr1: add 2 links [ph discord by famefgc, mx discord by tremolo3]
sfa3: updated 1 link [kyokuji]

IcDasNMs committed:
samsho2pe info

IcDasNMs committed:
changes until 2026-04-10

vsavj [h0llowsid]
flycast_gram2000 [crowspaceboy]

JordiRos committed:
Merge branch 'main' into main
